### DISCUSSION TITLE
import failed

### DISCUSSION THREAD

**Discussion by u007** [vikejs/vike#1710] [Discussion]
pages/+config.ts

```
import vikeVue from "vike-vue/config";
import type { Config } from "vike/types";
import Head from "../layouts/HeadDefault.vue";
import Layout from "../layouts/LayoutDefault.vue";

// Default config (can be overridden by pages)
export default {
  Layout,
  Head,
  // <title>
  title: "AAA BackOffice",
  extends: vikeVue,
} satisfies Config;

```

got error:
```
10:19:34 PM [vike][config][Wrong Usage] The import path '/Users/james/www/aaa/admin/layouts/HeadDefault.vue' in /pages/+config.ts couldn't be resolved: does '/Users/james/www/aaa/admin/layouts/HeadDefault.vue' exist?
```

the HeadDefault.vue does exists and its the default generated template

**Comment by brillout** [comment]
Can you provide a reproduction? What's your Node.js version?

**Reply by u007** [reply]
ok just realise its due to bunx --bun vite
if i run bunx vite, it works

**Reply by brillout** [reply]
:+1: Thanks for circling back on this. (PR welcome if you believe there is a easy workaround for this.)

**Reply by u007** [reply]
here is my repo
https://github.com/u007/vike-admin

**Reply by u007** [reply]
how can i help? could this be path issue with bun --bun ?

**Reply by brillout** [reply]
I ain't familiar with Bun so I'm not sure I can help much. That said, it's probably a known Bun bug, with maybe a workaround.
